composer.json

```diff
@@ -126,7 +126,7 @@
         "phpstan-config": "vendor/bin/phpstan analyse config --ansi --error-format symplify",
         "phpstan-fixtures": "vendor/bin/phpstan analyse -c phpstan-for-fixtures.neon --ansi --error-format symplify",
         "docs": [
-            "vendor/bin/rule-doc-generator generate rules --output-file build/rector_rules_overview.md --ansi --categorize 3",
+            "vendor/bin/rule-doc-generator generate rules --output-file build/rector_rules_overview.md --ansi --categorize 3 --skip-type \"Rector\\Core\\Rector\\AbstractCollectorRector\"",
```

Comment:
would this work?

```suggestion
            "vendor/bin/rule-doc-generator generate rules --output-file build/rector_rules_overview.md --ansi --categorize 3 --skip-type 'Rector\\Core\\Rector\\AbstractCollectorRector'",
```

Reply:
Yes, that works 👍 , thank you.

This PR need a patch on rule-doc-generator first:

- https://github.com/symplify/rule-doc-generator/pull/7